src/graphorm/index.ts

```diff
@@ -187,6 +187,28 @@ const obj = new GraphORM({
               .andWhere(`${childAlias}."sourceId" = "${parentAlias}".id`),
         },
       },
+      owners: {
+        select: (ctx, alias, qb) => {
+          return qb
+            .select('ARRAY_AGG(sm."userId")')
+            .from(SourceMember, 'sm')
+            .where(`sm."sourceId" = "${alias}".id`)
+            .andWhere(`sm.role = 'owner'`)
+            .limit(50); // limit to avoid huge arrays, most squads should fit into this
+        },
+        transform: nullIfNotLoggedIn,
+      },
+      moderators: {
+        select: (ctx, alias, qb) => {
+          return qb
+            .select('ARRAY_AGG(sm."userId")')
+            .from(SourceMember, 'sm')
+            .where(`sm."sourceId" = "${alias}".id`)
+            .andWhere(`sm.role = 'moderator'`)
+            .limit(50); // limit to avoid huge arrays, most squads should fit into this
```

Comment:
I think we can omit the comments.
```suggestion
            .limit(50);
```

Reply:
Hmm, since is not visible by first why this is I though comments are useful, I guess the changes will be linked to the PR where we have some info about it. @rebelchris WDYT?

Reply:
No strong feelings about it, comments like this can be useful as they are very specific and may cause confusion down the line.

Reply:
I would like to leave it if you don't mind. Just because it is specific beta feature. There is only one comment now since logic is extracted to `getMembersByRole` as Chris noted.